src/pages/edit-kit/edit-kit.ts

```diff
@@ -68,15 +69,36 @@ export class EditKitPage {
    * Creates or updates the kit, and creates and deletes kit models.
    */
   onSave(form: NgForm) {
-    if (this.action === Actions.add) {
-      this.layoutActions.showLoadingMessage(LoadingMessages.creatingKit);
-      this.kitsActions.createKit(form.value, this.modelsToCreate);
+    if (!this.modelsToCreate.length) {
+      let alert = this.alertCtrl.create({
+        title: 'No items in kit',
+        message: 'Please add at least one item to the kit',
+        buttons: [
+          {
+            text: 'Cancel',
+            role: 'cancel'
+          },
+          {
+            text: 'Add Item',
+            handler: () => {
```

Comment:
This could be shortened to `handler: () => this.onAddItem()` or `handler: this.onAddItem` if you don't need the `this` context.